### PR TITLE
Blog Page-Layout

### PR DESCRIPTION
**Pull request by suryaaa-g**
Bare-bones

### CODE REVIEW

File: css/blog-style.css [hunk 1]
```diff
@@ -90,11 +91,124 @@ body {
...
 
 .main-container {
-    margin-top: 4rem;
+    margin-top: 5rem;
 }
 
-.text-center {
+/*BLOG IMAGE AND TITLE BEGINS  */
+
+.Blog-Title {
```
Comment (rzmk) [inline css/blog-style.css:99]: Keep class names lowercase
```suggestion
.blog-title {
```

File: html/blog.html [hunk 2]
```diff
@@ -29,6 +32,86 @@
         </div>
     </header>
     <!--NAVIGATION BAR ENDS-->
+
+    <!--Main Content Begins-->
+    <div class="main-container">
+
+        <main>
+            <section class="Blog-Title">
```
Comment (rzmk) [inline html/blog.html:40]: Keep class names lowercase
```suggestion
            <section class="blog-title">
```

File: html/blog.html [hunk 3]
```diff
@@ -29,6 +32,86 @@
...
+                    <img src="../imgs/quote2.jpg" alt="popular" />
+                    <img src="../imgs/blog-cover.jpg" alt="popular" />
+                </div>
+                <div class="card">
+                    <h3>Follow Us</h3>
+                    <p>Some Text...</p>
+                </div>
+            </div>
+        </div>
+
```
Comment (rzmk) [inline html/blog.html:87]: Add main content ends comment
```suggestion
    <!--Main Content Ends-->
```

**Review by rzmk — CHANGES_REQUESTED** [review]
- Review the commented changes (accept suggested code).
- Upload all relevant image files to imgs folder.

**Comment by suryaaa-g** [comment]
done

**Review by rzmk — APPROVED** [review]
Good job! Blog structure complete!

**Pull request merged**